cobalt/network/switches.cc

```diff
@@ -42,6 +42,8 @@ const char kMaxNetworkDelayHelp[] =
 // Switch to disable use of the Quic network protocol.
 const char kDisableQuic[] = "disable_quic";
 
+const char kDisableInAppDial[] = "disable_in_app_dial";
```

Comment:
One small request: Please move this into `ENABLE_DEBUG_COMMAND_LINE_SWITCHES`. We don't want disabling it to be an allowed option on any production platforms.